config/database.yml

```diff
@@ -0,0 +1,9 @@
+development:
+  adapter: postgresql
+  host: localhost
+  database: tictactoe
+  encoding: unicode
+
+test:
+  adapter: postgresql
+  database: tictactoe_test
```

Comment:
You might have seen this feature already, but you can remove duplication in YAML with anchors:

https://confluence.atlassian.com/bitbucket/yaml-anchors-960154027.html